api/reference/metric_rollups.adoc

```diff
@@ -0,0 +1,72 @@
+
+[[metric-rollups]]
+== Metric Rollups
+
+Querying of Metric Rollups is made available via the following collection:
+
+[source,data]
+----
+/api/metric_rollups
+----
+
+Supported query parameters:
+
+[options=header,cols="2,4",width=80%]
+|======================
+| Parameter | Examples
+| resource_type | VmOrTemplate, Service
```

Comment:
Just noticed this - we may want to note that technically from the metric rollups collection, anything resource_type that has metric rollups is supported. This is because we use the `.rollups_in_range` method from `MetricRollup` itself.

Reply:
I second that! At first glance I thought that just VmOrTemplate or Service was supported.